github/github.go

```diff
@@ -28,9 +28,11 @@ import (
 )
 
 const (
+	packageVersion = "45.2.0"
```

Comment:
I couldn't find this defined anywhere else in the package code, so I added it here. I'm happy to move it elsewhere, or access it in some other way if there are good options out there.